modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js

```diff
@@ -12,10 +12,23 @@
  * details.
  */
 
+import ClayEmptyState from '@clayui/empty-state';
+import PropTypes from 'prop-types';
 import React from 'react';
 
-export default function TemplatesPanel() {
-    return (
-        <h1>Templates Panel</h1>
-    );
+export default function TemplatesPanel({items}) {
+	return items ? (
```

Comment:
For an empty array, we will need the ClayEmptyStat too. 
I think it's better to check the length too `items?.length`

Reply:
done!